gnovm/pkg/gnomod/pkg.go

```diff
@@ -5,40 +5,32 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 type Pkg struct {
```

Comment:
I think type Module would be better.
They are all modules, by virtue of having .mod files right?
This differentiates from PackageValue, and amino Package too.

Reply:
I initially had the same thought. 

But we don't seem to use word module anywhere. 
We only use the word `pkg` (and pkg with state = realm)